lib/service.dart

```diff
@@ -84,12 +102,16 @@ class ServiceConnectionManager {
       }
 
       service = _service;
```

Comment:
not a bug in this CL but parameter names should not start with _.
Fix that while you are here changing _service to service.

Reply:
Done.